Foundation/src/DirectoryWatcher.cpp

```diff
@@ -299,9 +299,8 @@ class LinuxDirectoryWatcherStrategy: public DirectoryWatcherStrategy
 		}
 		
 		Poco::Buffer<char> buffer(4096);
-		while (!_stopped.load(std::memory_order_relaxed))
-		{
-			fd_set fds;
+        while (!_stopped)
```

Comment:
Removed indent by accident.

Reply:
Yes, poco use TAB for indent, my editor format it to use space when edit the code.